packages/dashboard/src/components/queryEditor/iotSiteWiseQueryEditor/assetModelDataStreamExplorer/assetModelDataStreamExplorer.tsx

```diff
@@ -1,5 +1,5 @@
 import React, { useEffect, useState } from 'react';
-import { type IoTSiteWise } from '@aws-sdk/client-iotsitewise';
+import { AssetSummary, type IoTSiteWise } from '@aws-sdk/client-iotsitewise';
```

Comment:
should this whole line be a type import? 
`import type { AssetSummary, IoTSiteWise } from '@aws-sdk/client-iotsitewise';`